packages/react-vis/jest.setup.js

```diff
@@ -0,0 +1,18 @@
+/*eslint-env node*/
```

Comment:
If I need to use the `describe` blocks I am afraid I will have to do it manually. It is quite tedious and can take a while so I think we shouldn't block this PR for that.

As for using `it` instead of `test` imo it can improve readability when the tests are named properly. If we are to keep current names it would give something like: 

`it('Axis: Showcase Example - Even more Custom axes')`

which doesn't makes more sense than:

`test(Axis: Showcase Example - Even more Custom axes')` 

Even if we use the `describe` block, it would still read `it('Even more Custom axes')`

So I guess my take is the same for both questions: we would need to update a lot of stuff manually so it can be done in a second PR in order to merge this one quicker and get rid of the really verbose logs we have with current tests

Reply:
ya.. I didn't mean for it to block this review, just a general question.